install/operator/go.sum

```diff
@@ -414,7 +414,6 @@ github.com/stretchr/testify v1.3.0 h1:TivCn/peBQ7UY8ooIcPgZFpTNSz0Q2U6UrFlUfqbe0
 github.com/stretchr/testify v1.3.0/go.mod h1:M5WIy9Dh21IEIfnGCwXGc5bZfKNJtfHm1UVUgZn+9EI=
 github.com/stretchr/testify v1.4.0 h1:2E4SXV/wtOkTonXsotYi4li6zVWxYlZuYNCXe9XRJyk=
 github.com/stretchr/testify v1.4.0/go.mod h1:j7eGeouHqKxXV5pUuKE4zz7dFj8WfuZ+81PSLYec5m4=
-github.com/syndesisio/syndesis v0.3.6 h1:so/K4EGWjkXakPUuWGDceqGmQyuG6UYTehaw3JDTd3s=
```

Comment:
@lgarciaaco not sure where this went, it always disappears when I build locally, so I guess it should be gone